### PR TITLE
docs: logical concepts update

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: Lee Calcote <lee.calcote@layer5.io>

**Notes for Reviewers**

This PR is a docs chore

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit ddb327a6195285fc20fe78c33197ab8aa96fbd2f at: https://659387fbf7667f380f1db797--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f73f84fc6499c96d9365e8afd879e6e819c65b82 at: https://6593c8d4b4d60c62d02e61ed--meshery-docs-preview.netlify.app

File: docs/pages/concepts/logical/designs.md [hunk 1]
```diff
@@ -14,9 +14,37 @@ Like a Google Doc, Designs are your primary tool for collaborative authorship of
 
 <!-- ### Using Designs -->
 
+A Design consists of **Components** and/or **Patterns**. A Design is the deployable unit in Meshery. Designs are how the users can describe the desired infrastructure state.
+There cannot be two components with the same name within a Design. However, there can be two components with the same name in different Designs.
```
Comment (goldfishdolphin) [inline docs/pages/concepts/logical/designs.md:18]: The names of the components are unique in a Design.

File: docs/pages/concepts/logical/designs.md [hunk 2]
```diff
@@ -14,9 +14,37 @@ Like a Google Doc, Designs are your primary tool for collaborative authorship of
...
+
 ### Design Relationships and Restrictions
 
-- Access to Designs may be granted to one or more users.
+- Designs belong to one an only one Workspace at any given time. Designs may be transferred between Workspaces.
```
Comment (goldfishdolphin) [inline docs/pages/concepts/logical/designs.md:22]: Designs belong to only one Workspace at any given time. Designs may be transferred between Workspaces.

Comment (goldfishdolphin) [reply docs/pages/concepts/logical/designs.md:22]: @leecalcote, please suggest if "can" or "may" is appropriate for the list of Designs.

**Review by goldfishdolphin — APPROVED** [review]
Well described document.

File: docs/pages/concepts/logical/designs.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/concepts/logical/designs.md:18]: Simpler. Easier o understand. 🙂

File: docs/pages/concepts/logical/designs.md (hunk 2 above)
Comment (leecalcote) [reply docs/pages/concepts/logical/designs.md:22]: THank you. Suggestion accepted.

Comment (leecalcote) [reply docs/pages/concepts/logical/designs.md:22]: You're right that "can" is the more appropriate term here.

**Pull request merged**